synapse/lib/stormlib/graph.py

```diff
@@ -0,0 +1,185 @@
+import synapse.exc as s_exc
+import synapse.lib.config as s_config
+import synapse.lib.stormtypes as s_stormtypes
+
+gdefSchema = {
+    'type': 'object',
+    'properties': {
+        'iden': {'type': 'string', 'pattern': s_config.re_iden},
+        'name': {'type': 'string'},
```

Comment:
Does a empty string count as a valid name?

Reply:
probably shouldn't, i'll check it